enterprise/models.py

```diff
@@ -107,3 +110,56 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+def logo_path(instance, filename):
+    """
+    Delete the file if it already exist and returns the enterprise customer logo image path.
+
+    :param instance: EnterpriseCustomerBrandingConfiguration object
+    :param filename: file to upload
+    :return path: path of image file e.g. enterprise_logo/1/filename
+    """
+    name = os.path.join('enterprise_logo', str(instance.id), filename)
+    fullname = os.path.join(settings.MEDIA_ROOT, name)
+    if os.path.exists(fullname):
+        os.remove(fullname)
+    return name
+
+
+class EnterpriseCustomerBrandingConfiguration(TimeStampedModel):
+    """
+    Model that keeps track of enterprise branding configurations e.g. enterprise customer logo.
+
+    Fields:
+        enterprise_customer (ForeignKey[EnterpriseCustomer]): enterprise customer
+        logo (ImageField): enterprise customer image
+    """
+
+    enterprise_customer = models.OneToOneField(
+        EnterpriseCustomer,
+        blank=False,
+        null=False,
+        related_name='branding_configuration'
+    )
+    logo = models.ImageField(
+        upload_to=logo_path,
```

Comment:
Sorry, I could not get the point. Storing an image depends upon the value of `DEFAULT_FILE_STORAGE` in system [LN#503-505](https://github.com/edx/edx-platform/blob/master/lms/envs/aws.py#L503). As per my understanding, prod uploading the media / static assets on **AWS S3** and locally uses the `FileSystemStorage`. However if you are talking about the bucket_name for media contents then let me know is it ? then what will be the credentials to create a new bucket on S3 ?